cms/static/js/views/pages/group_configurations.js

```diff
@@ -1,11 +1,16 @@
 define([
     'jquery', 'underscore', 'gettext', 'js/views/pages/base_page',
-    'js/views/group_configurations_list', 'js/views/content_group_list'
+    'js/views/group_configurations_list', 'js/views/partition_group_list'
 ],
-function($, _, gettext, BasePage, GroupConfigurationsListView, ContentGroupListView) {
+function($, _, gettext, BasePage, GroupConfigurationsListView, PartitionGroupListView) {
     'use strict';
     var GroupConfigurationsPage = BasePage.extend({
         initialize: function(options) {
+            var currentScheme,
+                i;
+            this.COHORT_SCHEME = 'cohort';
+            this.ENROLLMENT_SCHEME = 'enrollment_track';
```

Comment:
There is no reason that this should be a class variable-- I will make it a local variable.